### ISSUE TITLE
[FEATURE_REQUEST] Widget for displaying locally stored images

### ISSUE THREAD

**Issue by RK62** [Lissy93/dashy#487]
### Is your feature request related to a problem? If so, please describe.

I would like to include images on my dashboard. 
This can be rendered Grafana graphics or just pictures from my last vacation.

### Describe the solution you'd like

The widget should display an image from the local /app/public/item-icons directory. Maybe it would be better to provide a separate directory for this. It would be great to be able to select a single image or play random images from a directory.

Thanks a lot for DASHY! I use it every day and find it really cool.

### Priority

Low (Nice-to-have)

**Comment by Lissy93** [comment]
Heya,
So you can actually already do this, with either of the dynamic widget options, you can make your own widget.

Foe example, using the [HTML Embedded Widget](https://github.com/Lissy93/dashy/blob/master/docs/widgets.md#html-embedded-widget), your config would be:

```yaml
- type: embed
  options:
    html: '<img src="https://dashy.lan/item-icons/media/plex.png" />'
```

**Comment by Lissy93** [comment]
We could add a dedicated widget just for displaying images, it would be really easy to do by extending from the `widgetMixin`.

For example, `src\components\Widgets\ImageWidget.vue`:

```vue
<template>
<div class="image-widget">
  <img :src="imagePath" class="embedded-image" />
</div>
</template>

<script>
import WidgetMixin from '@/mixins/WidgetMixin';
export default {
  mixins: [WidgetMixin],
  computed: {
    imagePath() {
      if (!this.options.imagePath) this.error('You must specify an imagePath');
      return this.options.imagePath;
    },
  },
};
</script>

<style scoped lang="scss">
.image-widget {
  img.embedded-image {
    max-width: 100%;
    margin: 0.2rem auto;
  }
}
</style>
```

Then just import that file in `WidgetBase.vue`, and maybe add it into the docs. More details on doing so is [available here](https://github.com/Lissy93/dashy/blob/master/docs/development-guides.md#building-a-widget).

**Comment by Lissy93** [comment]
I've implemented this in #488, Let me know how you get on with that

---

### Example

```yaml
- name: Image Example
  icon: far fa-image
  widgets:
  - type: image
    options:
      imagePath: https://picsum.photos/400
```
![image](https://user-images.githubusercontent.com/1862727/153879628-2195e5c7-713f-4537-9ba1-8f3e9f881437.png)

See the [Docs](https://github.com/Lissy93/dashy/blob/master/docs/widgets.md#image) for full usage instructions.

**Comment by RK62** [comment]
Thanks a lot, this works great!
In docker-compose I mount the directory 
`- ${SSDPROD}/dashy/media:/app/public/media`
and then can use 
```
      - type: image
        options:
           imagePath: /media/bobcat1.png
```
in Dashy to show it.
The given links on https://github.com/Lissy93/dashy/blob/master/docs/widgets.md#image are very helpful!
